src/pages/login/page.js

```diff
@@ -1,8 +1,14 @@
 import React from "react";
+import { browserHistory } from "react-router";
 import styles from "./style.css";
 
 
 export default class LoginPage extends React.Component {
+  
+  signUp(){
```

Comment:
I'm all for linting but also wary of including a particular .eslintrc file with opinionated rules in a generic skeleton app. Maybe some very minimal, sane defaults could suffice, and then it could be part of the travis build.